controllers/forgot_password.php

```diff
@@ -58,7 +58,10 @@
     	'MIME-Version' => $mime,
     	'Content-type' => $content);
     
-    $mail = $smtp->send($email, $headers, sprintf($email_forgot_password_body, $realname, $secret, $secret));
+    $email_body = sprintf($email_forgot_password_body, $realname, $secret, $secret);
+    $email_body = str_replace("<CODE>", $this->db->code, $email_body);
```

Comment:
no need to go through the db class, we already have this initialized on line 12
```suggestion
    $email_body = str_replace("<CODE>", $code, $email_body);
```